### PR TITLE
♻️ Adding CIA triad to PRs

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

I find it convenient to have the following reminders in the PRs for all the developers. Since we are not used to thinking about them.

Explanation was generated using chat-gpt generated the reply. Let's ensure the text makes sense.

Blank PRs will look like so:

Screenshot 2023-05-31 at 14 47 05

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -54,3 +54,20 @@ Modified from https://oschvr.com/posts/what-id-like-as-sre/
 - What are the resource limitations (CPU, RAM) expected for this service ?
 - Are all relevant variables documented and adjustable via environment variables (i.e. no hardcoded magic numbers) ?
 -->
+
+<!---
+Have you though about the CIA triad?
```
Comment (sanderegg) [inline .github/PULL_REQUEST_TEMPLATE.md:59]:
```suggestion
Have you thought about the CIA triad?
```

**Review by sanderegg — COMMENTED** [review]
ok... not sure I get the point... is this a new blaming mechanism?

**Comment by GitHK** [comment]
> ok... not sure I get the point... is this a new blaming mechanism?

No I don't see anything coming from blaming. The idea is to have a checklist of things. Usually I forget about these. My idea is to help others as well.

**Comment by GitHK** [comment]
@sanderegg I see now that the wording could be improved to avoid any blaming.

**Review by pcrespov — COMMENTED** [review]
The idea is good but this addition still sounds very generic to me. Why don't we present these ideas rationale with example in a dev-talk and then afterwards include them here in a succinct way?

We have been optimizing (and reducing) the text in the template with time but IMO Devops checklist is getting longer and longer at a risk of not being useful at all.

**Pull request closed without merging**